content/BindingMethods/BindingMethods.q.md

```diff
@@ -0,0 +1,1601 @@
+
+- `$data`
+
+- `PDO::PARAM_STR`
+
+- `:min_birth_date`
+
+- `$data['min_birth_date']`
+
+- `(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR)`
+
+
+/// type=SS, answer=[4]
+
+In `student.php`, which of the following represents the value to bind in the SQL statement on line 12?
+
+- `$data`
+
+- `PDO::PARAM_STR`
+
+- `:min_birth_date`
+
+- `$data['min_birth_date']`
+
+- `(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR)`
+
+
+/// type=SS, answer=[3]
+
+In `student.php`, which argument represents the explicit data type in the statement `$pstmt->bindParam(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR);` on line 12?
+
+- `$data`
+
+- `min_birth_date`
+
+- `PDO::PARAM_STR`
+
+- `:min_birth_date`
+
+- `$data['min_birth_date']`
+
+
+/// type=MS, answer=[1,4]
+
+In `student.php`, which statements correctly describe `(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR)` on line 12?
+
+- It represents the arguments of the `bindParam()` method.
+
+- It contains the key, array variable, data type of the parameter.
+
+- It contains the variable, value, and data type of the parameter.
+
+- It represents the parameter name, value, and data type of the parameter.
+
+- It contains the key, array element, and constant data type of the parameter.
+
+
+/// type=SS, answer=[4]
+
+In `student.php`, what does the `bindParam()` method do on line 12?
+
+- It assigns the value `1999-01-01` to the named placeholder `:min_birth_date`.
+
+- It updates the value `1990-01-01` to the named placeholder as parameter `:min_birth_date`.
+
+- It modifies the value `2000-01-01` to the named placeholder as parameter `:max_birth_date`.
+
+- It binds the value of `$data['min_birth_date']` to the corresponding named placeholder `:min_birth_date`.
+
+- It binds the value of `$data['max_birth_date']` to the corresponding named placeholder `:max_birth_date`.
+
+
+/// type=MS, answer=[2,3,5]
+
+Which statements correctly describe `$pstmt->bindParam(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR);` on line 12?
+
+- It executes the SQL statement.
+
+- It returns the boolean value `true`.
+
+- It calls the `bindParam()` method of the `$pstmt` object.
+
+- It assigns the named placeholder `:min_birth_date` to `$pstmt`.
+
+- It binds the value of `$data['min_birth_date]` to the named placeholder `:min_birth_date` in the SQL statement.
+
+
+/// type=SS, answer=[4]
+
+In `student.php`, which of the following represents the value to bind in the SQL statement on line 13?
+
+- `$data`
+
+- `:max_birth_date`
+
+- `max_birth_date`
+
+- `$data['max_birth_date']`
+
+- `(':min_birth_date', $data['min_birth_date'], PDO::PARAM_STR)`
+
+
+/// type=SS, answer=[4]
+
+In `student.php`, which of the following represents the parameter name in the statement `$pstmt->bindParam(':max_birth_date', $data['max_birth_date'], PDO::PARAM_STR);` on line 13?
```

Comment:
same here... please verify on which term to use? Is it `parameter name` or `paramater identifier`. Thanks.